hr_holidays_leave_overlap/models/hr_holidays.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2018 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from datetime import timedelta
```

Comment:
Follow the OCA Guidelines to order the imports: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports